src/main/java/parking/repositories/LotsRepositoryImpl.java

```diff
@@ -9,11 +9,13 @@
 import parking.beans.document.Account;
 import parking.beans.document.AvailablePeriod;
 import parking.beans.document.ParkingLot;
-import parking.beans.request.ParkingNumberRequest;
-import parking.beans.request.RecallSingleParking;
-import parking.beans.request.SetUnusedRequest;
 import parking.helper.ToolHelper;
+import parking.utils.ParkingType;
+import sun.util.calendar.CalendarSystem;
```

Comment:
warning: CalendarSystem is internal proprietary API and may be removed in a future release
import sun.util.calendar.CalendarSystem;